ant/toplevel.xml

```diff
@@ -959,6 +960,16 @@ Type "ant -p" for a list of targets.
     <ant dir="components/xsd-fu" target="xsd-fu.clean"/>
   </target>
 
+  <target name="generate-sources"
+    description="generate sources using xsd-fu"
+    depends="ome-xml-src, metadata-src">
+    <mkdir dir="generated-sources"/>
+	<copy todir="generated-sources">
+	    <fileset dir="components/ome-xml/target/generated-sources"/>
+	</copy>
+    <ant dir="components/ome-xml" target="generate-source"/>
```

Comment:
Isn't this ant call redundant since it's already run by the target dependencies?